app/models/task.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :integer          not null, primary key
+#  name        :string
+#  description :string
+#  progress    :integer          default(0)
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  project_id  :uuid
+#
+# Indexes
+#
+#  index_tasks_on_project_id  (project_id)
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  validates_presence_of :name
+  validates_presence_of :description
+  validates_presence_of :project_id
```

Comment:
Good addition. Since this is Rails and ActiveRecord, an even better option might be:
`validates_presence_of :project`

Your validation just validates that there is a `project_id` on the task.
Validating the presence of `project` ensures that not only is there ` project_id` on the task but it is the id of an actual project in the database.